packages/flutter_tools/test/general.shard/build_system/targets/dart_test.dart

```diff
@@ -159,6 +159,7 @@ flutter_tools:lib/''');
       depFilePath: anyNamed('depFilePath'),
       packagesPath: anyNamed('packagesPath'),
       mainPath: anyNamed('mainPath'),
+      linkPlatformKernelIn: true,
```

Comment:
Nit: maybe anyNamed('linkPlatformKernelIn'), as this test case doesn't even verify aot and buildMode?

Reply:
done